assignment.js

```diff
@@ -112,6 +140,45 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
+const playerAChoice = 'scissors'
```

Comment:
@nataliekk  nice job covering all of the possible outcomes. 

The approach that you're using involves nested `if` statements and while it works, you will want to avoid nesting if possible. The reason is that nesting can add complexity to your code which makes things more challenging to debug. However, in this case the code is still easy to follow.